src/components/package/create/package-create.js

```diff
@@ -80,11 +64,11 @@ class PackageCreate extends Component {
           <PaneHeader
             paneTitle={<FormattedMessage id="ui-eholdings.package.create.custom" />}
             actionMenuItems={actionMenuItems}
-            firstMenu={historyState && historyState.eholdings && (
+            firstMenu={onCancel && (
```

Comment:
Looks like `onCancel` is always provided to these `*-create` views from the create routes. So we either don't need this `onCancel &&` or the logic in those routes is meant to sometimes return a falsey value (it uses the ternary operator, so it always returns either `history.goBack` or `history.push`).

This applies to this `package-create` route and also the `title-create` route.

Reply:
Changed to sometimes return a falsey value. We don't want there to be a back button if there's nowhere to go back to.